addons/pos_discount/models/res_config_settings.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    # pos.config fields
+    pos_discount_pc = fields.Float(related='pos_config_id.discount_pc', readonly=False)
+    pos_discount_product_id = fields.Many2one('product.product', compute='_compute_pos_discount_product_id', store=True, readonly=False, pos='discount_product_id')
+
+    @api.depends('company_id', 'pos_module_pos_discount', 'pos_config_id')
+    def _compute_pos_discount_product_id(self):
+        def_product = self.env.ref("point_of_sale.product_product_consumable", raise_if_not_found=False)
```

Comment:
ault?

But this crashes when the record is missing. Need to consider it.

Reply:
I talked to @pimodoo and we decided to allow empty value for `discount_product_id` so that installation of `pos_discount` addon is not blocked when `def_product` is missing. But we block when opening a new session. I'll fix this logic to consider the situation where `def_product` is `None`.

Reply:
> ault?

def-ault `def_product` instead of `default_product` seems like a minimal gain in convenience and a large loss in readability.